File: codigos/1 - estrutura sequencial/1019 - Conversao de Tempo/Main.clj
(defn main []
    (let [total (read)
          horas (int (/ total 3600))
          minutos (int (/ (- total (* horas 3600)) 60))
          segundos (- total (* horas 3600) (* minutos 60))]
        (println (str horas ":" minutos ":" segundos))))

(main)

File: codigos/1 - estrutura sequencial/1019 - Conversao de Tempo/Main.dart
import 'dart:io';

main(){
  int total = int.parse(stdin.readLineSync()!);
  
  int horas = total ~/ 3600;
  int minutos = (total % 3600) ~/ 60;
  int segundos = total % 60;

  stdout.writeln("$horas:$minutos:$segundos");
}